csharp/docfx/docfx.json

```diff
@@ -5,13 +5,14 @@
         {
           "files": ["src/*/*.csproj"],
           "exclude": [
```

Comment:
As I understand it, it's about suppressing the csproj, which does not help with Ice.Internal / Ice.UtilInternal, since they don't have a csproj.

Reply:
For some reason I thought this could reference source files as well.